randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -352,7 +352,12 @@ Extra - asset_id: collision_camera_003
           Before Artaria - Proto EMMI Blob and Can Slide
           Lay Small Bomb or Shoot Beam
   > East of Blob
-      Morph Ball and After Artaria - Proto EMMI Blob
+      All of the following:
+          After Artaria - Proto EMMI Blob
+          Any of the following:
+              Morph Ball
+              # Hand wave, increases variety of first accessible pickup.
```

Comment:
could you explain in further detail in this comment why this change is safe and correct? just since it's very unintuitive at a glance

Reply:
Is this good? 
![image](https://github.com/user-attachments/assets/b4c759fe-0cf8-4717-9e23-ed20fab64d59)

Reply:
Yes, perfect